module/Finna/src/Finna/View/Helper/Root/RecordDataFormatterFactory.php

```diff
@@ -837,7 +837,7 @@ protected function getDefaultCoreFields()
             ]
         );
         $setTemplateLine(
-            'Date', 'getUnitDates', 'data-lines-with-detail.phtml'
+            'Dates', 'getUnitDates', 'data-lines-with-detail.phtml'
```

Comment:
Voisi olla parempi laittaa tähän jokin avain, jota ei ole käännöksissä, ja palauttaa oikea avain labelFunction:lla (ks. esim. https://github.com/NatLibFi/NDL-VuFind2/blob/dev/module/Finna/src/Finna/View/Helper/Root/RecordDataFormatterFactory.php#L161). Näin ei tarvitsisi tuota tuplakäännöstä.

Reply:
Itse asiassa senhän saa meidän templaateissa vielä helpommin laittamalla contextiin title-kentän tähän tapaan: https://github.com/NatLibFi/NDL-VuFind2/blob/dev/module/Finna/src/Finna/View/Helper/Root/RecordDataFormatterFactory.php#L457